ansible/roles/set-repositories/tasks/main.yml

```diff
@@ -18,3 +18,7 @@
     - name: Configure Repository File
       import_tasks: ./file-repos.yml
```

Comment:
prefer `include_tasks` here so the tasks are not imported.

`import_tasks` + a false `when` ==  all tasks are imported and the condition is copied to all the imported tasks == logs are full of `skipped` tasks.

`include_tasks` + a false `when` ==  the include is skipped, nothing is included.

Reply:
These have been switched...thanks for the tip.